### PR TITLE
Fixed Channel Labels

### PR DESCRIPTION
**Pull request by alviriseup**
Fixed Interaction by Channel labels and chart

### CODE REVIEW

**Review by paulobernardoaf — APPROVED** [review]

**Comment by paulobernardoaf** [comment]
Is it done to be merged and deployed @alviriseup? If so, could you please update the CHANGELOG.md?

**Comment by alviriseup** [comment]
The CHANGELOG.md has been updated @paulobernardoaf .

**Comment by alviriseup** [comment]
@paulobernardoaf please review for merging and deploying.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@ v1.5.5
 
 Frontend Changes: 
 
+- Added WhatsApp, Viber, and VK to the Interactions by Channel chart list
```
Comment (paulobernardoaf) [inline CHANGELOG.md:5]: The changelog modification should create a new version `v1.5.6` with the modifications for this version. The added lines should not be in `v1.5.5`. It should follow the pattern from the earlier versions. @alviriseup

**Review by paulobernardoaf — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alviriseup) [reply CHANGELOG.md:5]: Noted, and updated the CHANGELOG.md accordingly. @paulobernardoaf

**Review by paulobernardoaf — APPROVED** [review]

**Pull request merged**